apps/web/src/components/AddressInputSelect/index.tsx

```diff
@@ -16,7 +16,7 @@ interface AddressInputSelectProps extends BoxProps {
   onAddressClick: (value: string) => void
 }
 
-const SubMenu = styled.div<{ isOpen: boolean }>`
+const SubMenu = styled.div<{ $isOpen: boolean }>`
```

Comment:
Is it because the new version `style-component` so in front props name need to add `$` ?

Reply:
with `$` the props will not pass to the DOM directly